### DISCUSSION TITLE
I installed Coqui-ai using Pinokio, but I don`t know how to start it.

### DISCUSSION THREAD

**Discussion by DieserBobby** [coqui-ai/TTS#3237] [General Q&A]
My http://127.0.0.1:7860/ is not connecting to the program.
![Coqui XTTS](https://github.com/coqui-ai/TTS/assets/63121369/4fbc9631-369e-470c-9c02-2938af9bc08e)
Can anybody give me the wow-effekt please?
How do I start? or is something missing?
![Coqui XTTS2](https://github.com/coqui-ai/TTS/assets/63121369/adab06b4-8368-4c21-be6a-59bc5ebd9915)

actually it looks like running... and usually my localhost works fine ( e.g. with 1111).
But in this case... localhost in chrome is not connecting...

**Comment by CLICKEARNER** [comment]
started and working on windows 10 but cant find my language hindi in pinokio version

**Reply by DieserBobby** [reply]
started over pinokio?

**Comment by Poliversia** [comment]
Same problem here. Any answer whatsoever?

It is showing this message down:
 (3/6) shell.enter
{"message":"python app.py","on":[{"event":"/(http://[0-9.:]+)/","return":"{{event.matches[0][1]}}"}]}

What is this supposed to mean. Installation has not been completed?

There are pending instances? But is not downloading anything else.
